proto2ros/proto2ros/utilities.py

```diff
@@ -47,7 +47,7 @@ def pairwise(iterable: Iterable[Any]) -> Iterable[Tuple[Any, Any]]:
     """Yields an iterable over consecutive pairs."""
     a, b = itertools.tee(iterable)
     next(b, None)
-    return zip(a, b)  # noqa: B905
```

Comment:
I think you need the comment since this intentional doesn't use the strict parameter?

Reply:
It got removed by `ruff` or `black`. Not sure. I think it's not required in Python 3.8.

Reply:
Yeah, apparently `strict` was added in 3.10...